tests/data/product_stability/ubuntu2004.yml

```diff
@@ -68,7 +68,7 @@ profiles_root: ./profiles
 reference_uris:
   anssi: https://cyber.gouv.fr/sites/default/files/document/linux_configuration-en-v2.pdf
   app-srg: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=application-servers
-  app-srg-ctr: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=container-platform
+  app-srg-ctr: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=app-security
```

Comment:
this one is not really needed as this file should have been removed.
I did this in PR #13768

Reply:
Rebased after #13768 was merged.